apps/client/src/dashboard-list/dashboard-list.page.spec.tsx

```diff
@@ -0,0 +1,23 @@
+import { screen } from '@testing-library/react';
+
+import { renderWithRouter } from '../test/render-with-router';
+
+describe('<DashboardListPage />', () => {
+  test('as a user, I want to view a list of my dashboards', () => {
```

Comment:
I love this test naming convention. I feel we should make this a standard practice.

Reply:
Thanks man! When I write test cases like user stories, I feel much more confident that I tested the right things. Would love to consider making it the standard. :)